doc/classes/ArrayMesh.xml

```diff
@@ -202,6 +202,9 @@
 			<param index="1" name="offset" type="int" />
 			<param index="2" name="data" type="PackedByteArray" />
 			<description>
+				Updates the vertex buffer of this mesh's surface with the given [param data]. The expected data per vertex is 12 or 8 bytes (4 bytes per float, 3 floats per [Vector3] and 2 floats per [Vector2]) depending on if the mesh is using [Vector3] or [Vector2] vertices. This value can be determined with [method mesh_surface_get_format_vertex_stride].
```

Comment:
```suggestion
				Updates the vertex buffer of this mesh's surface with the given [param data]. The expected data per vertex is 12 or 8 bytes (4 bytes per float, 2 floats per [Vector2], and 3 floats per [Vector3]) depending on if the mesh is using [Vector3] or [Vector2] vertices. This value can be determined with [method mesh_surface_get_format_vertex_stride].
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything). Also makes more sense to order in ascending size IMO